daprdocs/content/en/operations/configuration/use-custom-certificates.md

```diff
@@ -0,0 +1,90 @@
+---
+type: docs
+title: "How-To: Use custom certificates in the Dapr sidecar"
+linkTitle: "Use custom certificates"
+weight: 6500
+description: "Configure the Dapr sidecar container to trust custom certificates"
+---
+
+The Dapr sidecar can be configured to trust custom certificates for communicating with external services. This is useful in scenarios where a self-signed certificate needs to be trusted, e.g., using the HTTP binding, configuring an outbound proxy for the sidecar, etc.
+
+{{< tabs Self-hosted Kubernetes >}}
+
+{{% codetab %}}
+
+When the sidecar is running as a process, certificates must be installed on the host operating system. 
+
+When the sidecar is running as a container,
+1. Certificates must be available to the sidecar container. This can be configured using volume mounts.
+1. The environment variable `SSL_CERT_DIR` must be set in the sidecar container, pointing to the directory containing the certificates.
+1. Additionally for Windows containers, the container needs to be run with administrator privileges. This is required by the Windows entrypoint script to install the certificates.
+
+This is an example using Docker compose:
+```yaml
+version: '3'
+services:
+  dapr-sidecar:
+    image: "daprio/daprd:edge"
+    command: [
+      "./daprd",
+     "-app-id", "myapp",
+     "-app-port", "3000",
+     ]
+    volumes:
+        - "./components/:/components"
+        - "./certificates:/certificates" # (STEP 1) Mount the certificates folder to the sidecar container
+    environment:
+      - "SSL_CERT_DIR=/certificates" # (STEP 2) Set the environment variable to the path of the certificates folder
+    user: ContainerAdministrator # (STEP 3) Optional, required for Windows containers
```

Comment:
I would change this to be commented by default. Then add before a comment that says “# Uncomment this for Windows containers”